packages/react-components/source/scss/library/components/forms/_input.scss

```diff
@@ -6,7 +6,7 @@
   border-radius: $puppet-common-border-radius;
   box-sizing: border-box;
   outline: none;
-  padding: 5px 7px;
+  padding: 7px;
```

Comment:
I think what you want is to change this to `3px 7px`, to maintain the 32px height with the change from 20px line height to 24px